lib/commands/connect/export.js

```diff
@@ -15,17 +15,21 @@ module.exports = {
   needsApp: true,
   needsAuth: true,
   run: cli.command(co.wrap(function* (context, heroku) {
-    let connection = yield api.withConnection(context, heroku);
-    let url = '/api/v3/connections/' + connection.id + '/actions/export';
-    let response = yield api.request(context.auth.password, 'GET', url)
+    let connection, response;
```

Comment:
heh, I was going to say 3 is most unixy or I maybe option 4) print to stdout unless -o filename is specified.  Mostly thinking about the quick and dirty ALM use case, later is fine.